x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/processors_tree/components/drop_zone_button.tsx

```diff
@@ -7,39 +7,61 @@
 import { i18n } from '@kbn/i18n';
 import React, { FunctionComponent } from 'react';
 import classNames from 'classnames';
-import { EuiButtonIcon, EuiFlexItem } from '@elastic/eui';
+import { EuiButtonIcon, EuiToolTip } from '@elastic/eui';
 
 export interface Props {
+  isVisible: boolean;
   isDisabled: boolean;
   onClick: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
   'data-test-subj'?: string;
 }
 
-const MOVE_HERE_LABEL = i18n.translate('xpack.ingestPipelines.pipelineEditor.moveTargetLabel', {
-  defaultMessage: 'Move here',
-});
+const moveHereLabel = i18n.translate(
+  'xpack.ingestPipelines.pipelineEditor.dropZoneButton.moveHereToolTip',
+  {
+    defaultMessage: 'Move here',
+  }
+);
+
+const cannotMoveHereLabel = i18n.translate(
```

Comment:
I wonder whether things would feel simpler to the user to just be allowed to move the processor anywhere, even if it's just to "drop" it back in place. If you click move accidentally and want to get out of that mode, would this be convenient?

Put another way: are we helping the user by telling them they can't move the processor above or below where it currently is? Technically, they _could_ move the processor there -- it just wouldn't change position. If a user did this, would that experience be so disorienting that we'd want to prevent it?

I'm just asking these questions aloud, I don't have any answers. :)

Reply:
Hm, that is an interesting point! And kind of philosophical 😄 . If something is moved, but its position did not change did it move?

I gave this some thought; the way I think about it is "wouldn't change position" is equivalent to not moving and this to me is the noop described by cannot move.

The primary goal I think this serves at the moment is giving us a way to make drop zones more discoverable (and the entire drop mode).